governance/votes/Executive vote - March 5, 2021.md

```diff
@@ -0,0 +1,78 @@
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it can only be executed between 14:00 and 21:00 UTC, Monday - Friday. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots caused by auction parameter changes.
+
+If this executive proposal does not pass within **30 days** then it will expire and can no longer have any effect on the Maker Protocol. 
+
+---
+
+## Proposal Details
+
+### Increase ETH-A Target Available Debt
+
+As per this [poll](https://vote.makerdao.com/polling/QmXKFm9n?network=mainnet#poll-detail) and the associated forum [signal](https://forum.makerdao.com/t/signal-request-increase-eth-a-dc-iam-gap/6603), the [Target Available Debt](https://community-development.makerdao.com/en/learn/governance/module-dciam/) (`gap`) will be increased from 30 million DAI to 80 million DAI for the ETH-A vault type if this executive proposal passes.
+
+### Collateral Auction Parameter Updates
+
+As per this [poll](https://vote.makerdao.com/polling/QmPyvEzd?network=mainnet) and the associated proposal [thread](https://forum.makerdao.com/t/flip-auctions-parameter-adjustment-proposal/6732) multiple auction parameters will be changed if this executive proposal passes.
+
+In addition to those changes listed in the poll, the Minimum Bid Increment (`beg`) of UNIV2DAIUSDT-A will also increase from 3% to 5%. This was an accidental omission from the poll due to the onboarding timeline of UNIV2DAIUSDT-A. It has been included here because it keeps all UniswapV2 liquidity pool token vault types with consistent auction parameters.
+
+### Onboarding RWA-001
+
+As per this [poll](https://vote.makerdao.com/polling/QmSqXVUQ?network=mainnet#poll-detail), this executive proposal represents the latest step in onboarding RWA-001 (6s Capital). If this executive proposal passes the RWA001-A vault type will be added using RWA001 (6s Capital) as collateral with preliminary, non-final parameters for testing purposes.
+
+The preliminary parameters for RWA-001 (6s Capital) are:
+
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 3%
+* [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 1000 DAI
+
+Please review the following forum threads containing information about RWA-001 (6s Capital) to inform your position before voting.
```

Comment:
Does it make sense to add MIP21 here? https://forum.makerdao.com/t/mip21-real-world-assets-off-chain-asset-backed-lender/3917